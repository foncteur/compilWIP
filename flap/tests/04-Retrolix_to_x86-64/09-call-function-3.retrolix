external printf

def print_three_int ()
local a1, a2, a3:
  a1: a1 <- copy %rdi;
  a2: a2 <- copy %rsi;
  a3: a3 <- copy %rdx;
  l10: %rdi <- copy "%d %d %d\n";
  l11: %rsi <- copy a1;
  l12: %rdx <- copy a2;
  l13: %rcx <- copy a3;
  l14: %rax <- copy 0;
  l15: printf();
  l16: ret;
end

globals (x)
  l01: %rdi <- copy 37;
  l02: %rsi <- copy 73;
  l03: %rdx <- copy 42;
  l04: print_three_int();
  l05: exit;
end