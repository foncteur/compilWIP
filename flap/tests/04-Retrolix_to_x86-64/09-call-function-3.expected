Process exited with status exited(0).
STDOUT:
37 73 42

STDERR:



